packages/jellyfish-api-core/__tests__/category/account.test.ts

```diff
@@ -479,4 +480,46 @@ describe('masternode', () => {
       })
     })
   })
+
+  describe('utxosToAccount', () => {
+    let address1: string
+    let address2: string
+
+    beforeAll(async () => {
+      address1 = await container.call('getnewaddress')
+      address2 = await container.call('getnewaddress')
+    })
+
+    it('should utxosToAccount', async () => {
+      const payload: UtxosToAccountPayload = {}
+      // NOTE(jingyi2811): Only support sending utxos to DFI account.
+      payload[address1] = '5@DFI'
+      payload[address2] = '5@DFI'
+
+      const data = await client.account.utxosToAccount(payload)
+
+      expect(typeof data).toBe('string')
+      expect(data.length).toBe(64)
+    })
+
+    it('should utxosToAccount with utxos', async () => {
+      const payload: UtxosToAccountPayload = {}
+      // NOTE(jingyi2811): Only support sending utxos to DFI account.
+      payload[address1] = '5@DFI'
+      payload[address2] = '5@DFI'
```

Comment:
Localize your address and remove beforeAll since you are not using it for any setup process. Aka, there is no reason for it not to be inside the `it` block. Unnecessary use of `beforeAll` when you are not actually setting up any condition that you need for `it`.

```ts
      payload[await container.getNewAddress()] = '5@DFI'
      payload[await container.getNewAddress()] = '5@DFI'
```